saleor/graphql/checkout/types.py

```diff
@@ -590,7 +592,9 @@ def resolve_delivery_method(root: models.Checkout, info: ResolveInfo):
             CheckoutInfoByCheckoutTokenLoader(info.context)
             .load(root.token)
             .then(
-                lambda checkout_info: checkout_info.delivery_method_info.delivery_method
+                lambda checkout_info: unwrap_lazy(
+                    checkout_info.delivery_method_info
+                ).delivery_method
```

Comment:
Nitpick, but this does not have any effect. The lazy object is already evaluated when you access `.delivery_method`, which is subsequently returned.